docs/en_US/Tutorial/QuickStart.md

```diff
@@ -133,31 +153,37 @@ trial:
   gpuNum: 0
 ```
 
-Note, **for Windows, you need to change the trial command from `python3` to `python`**.
+```eval_rst
+.. Note:: If you are planning to use remote machines or clusters as your :doc:`training service <../TrainingService/SupportTrainingService>`, to avoid too much pressure on network, we limit the number of files to 2000 and total size to 300MB. If your codeDir contains too many files, you can choose which files and subfolders should be excluded by adding a ``.nniignore`` file that works like a ``.gitignore`` file. For more details on how to write this file, see the `git documentation <https://git-scm.com/docs/gitignore#_pattern_format>`_.
```

Comment:
suggest to give a single example for this .nniignore file, but not in this file.